_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -93262,20 +93265,15 @@
 	name = "\improper AI Satellite Service"
 	})
 "dle" = (
-/obj/machinery/the_singularitygen{
-	anchored = 1
-	},
 /obj/effect/decal/warning_stripes/west,
+/obj/item/wrench,
 /turf/simulated/floor/plating/airless,
 /area/space/nearstation)
 "dlf" = (
-/obj/item/wrench,
+/obj/item/radio/beacon/engine/tesling,
 /turf/simulated/floor/plating/airless,
 /area/space/nearstation)
 "dlg" = (
-/obj/machinery/the_singularitygen/tesla{
```

Comment:
Maybe also remove the singulogen in secure storage?

Reply:
This is meant to replace the containment-available engines, not storage ones. This PR already technically shrinks the available engine count by one, so it may be too much to remove the storage one as well.

Reply:
I never got why there was even a spare singulogen in storage. Why isnt there a tesla one as well?